src/igprof-navigator

```diff
@@ -257,8 +257,8 @@ def summary(database, out, cumulative):
 
 def column(value, isInt):
   if isInt:
-    return "<td>%s</td>" % "{}".format(int(value), True)
-  return "<td>%s</td>" % "{}".format( float(value), True)
+      return "<td>%s</td>" % "{0:.d}".format(int(value), True)
```

Comment:
I think the whole script uses 2 spaces, not 4.

Reply:
Changed to 2 spaces.